Dockerfile

```diff
@@ -19,46 +46,41 @@ ENV PYTHONUNBUFFERED=1\
     PIP_NO_CACHE_DIR=off \
     # Save runtime because it is not look for updating pip version
     PIP_DISABLE_PIP_VERSION_CHECK=on \
-    PIP_DEFAULT_TIMEOUT=100
-
-USER root
+    PIP_DEFAULT_TIMEOUT=100 \
+    # Disable poetry interaction
+    POETRY_NO_INTERACTION=1 \
```

Comment:
I can't find this setting in the documentation: https://python-poetry.org/docs/configuration/